lib/resources/example_json/r4_examples_service_request.rb

```diff
@@ -0,0 +1,157 @@
+          "code": "2515",
+          "display": "Patient Care",
+          "userSelected": true
+        }, {
+          "system": "http://snomed.info/sct",
+          "code": "108252007",
+          "display": "Laboratory procedure"
+        }],
+        "text": "Patient Care"
+      }],
+      "priority": "routine",
+      "code": {
+        "coding": [{
+          "system": "http://www.nlm.nih.gov/research/umls/rxnorm",
+          "code": "854986",
+          "display": "Perindopril Erbumine 2 MG Oral Tablet [Aceon]",
+          "userSelected": false
+        }],
+        "text": "Resuscitation Status"
+      },
+      "orderDetail": [
+        {
+          "text": "03/24/20 19:05:00, Full Resuscitation"
+        }
+      ],
+      "quantityQuantity": {
+        "value": 3.0
+      },
+      "subject": {
+        "reference": "Patient/1316020",
+        "display": "Lagunov, Sameera"
+      },
+      "encounter": {
+        "reference": "Encounter/12157884"
+      },
+      "occurrenceDateTime": "2020-03-25T00:05:00.000Z",
+      "asNeededCodeableConcept": {
+        "coding": [
+          {
+            "system": "https://fhir.cerner.com/0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca/codeSet/4005",
+            "code": "664912",
+            "display": "inflammation",
+            "userSelected": true
+          },
+          {
+            "system": "http://snomed.info/sct",
+            "code": "1415005",
+            "display": "Inflammation of lymphatics"
+          }
+        ],
+        "text": "inflammation"
+      },
+      "authoredOn": "2020-03-25T00:05:30.000Z",
+      "requester": {
+        "reference": "Practitioner/1626008",
+        "display": "McCready, Tim"
+      },
+      "performer": [
+        {
+          "reference": "Practitioner/664912",
+          "display": "Dr. Spoc"
+        },
+        {
+          "reference": "Practitioner/1744062",
+          "display": "Dr. Dre"
+        }
+      ],
+      "locationReference": [
+        {
+          "reference": "Location/309348",
+          "display": "Mayo Clinic"
+        }
+      ],
+      "reasonCode": [
+        {
+          "text": "Fracture"
+        }
+      ],
+      "bodySite": [
+        {
+          "coding": [
+            {
+              "system": "https://fhir.cerner.com/0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca/codeSet/1028",
+              "code": "309344",
+              "display": "Arm L",
+              "userSelected": true
+            },
+            {
+              "system": "http://snomed.info/sct",
+              "code": "362728000",
+              "display": "entire right upper extremity (body structure)"
+            }
+        ],
+        "text": "Arm L"
+        }
+      ],
+      "note": [
+        {
+          "authorReference": {
+            "reference": "Practitioner/1",
+            "display": "Some, Name"
+          },
+          "time": "2014-10-10T15:25:45.000Z",
+          "text": "A comment"
+        }
+      ],
+      "text": {
```

Comment:
Can we move this field to after 'meta' to rightly reflect how the data is returned on a transaction. 
We would always want to preserve the order of the fields as per the hl7 documentation and so it would be nice if we can reflect the very same in our example.